test/certificate_test.go

```diff
@@ -0,0 +1,387 @@
+package test
```

Comment:
Would this be more canonically located as certs_test in the certs package? I'm curious why it's here.

Reply:
No good reason. IIRC, it was because:

1. The certificates used to live in the gpbft package.
2. I didn't want to import the simulation.

Or something like that.

I'll move it in #293 (where I'll also be adding more tests).